pandas/errors/__init__.py

```diff
@@ -91,6 +91,42 @@ class PerformanceWarning(Warning):
     """
 
 
+class PandasChangeWarning(Warning):
+    """
+    Warning raised for any an upcoming change.
+    """
```

Comment:
Not in love with this name, but `PandasDeprecationWarning` is taken below. Bikeshedding here is most welcome!

Reply:
We could also make this internal if we want to not have it be public.